src/network/net_modem.c

```diff
@@ -1,5 +1,22 @@
 
-/* TODO: SLIP support. */
+/*
+ * 86Box    A hypervisor and IBM PC system emulator that specializes in
+ *          running old operating systems and software designed for IBM
+ *          PC systems and compatibles from 1981 through fairly recent
+ *          system designs based on the PCI bus.
+ *
+ *          This file is part of the 86Box distribution.
+ *
+ *          Hayes AT-compliant modem emulation.
+ *
+ *
+ *
+ * Authors: Cacodemon345
+ *          The DOSBox Team
+ *
+ *          Copyright 2024 Cacodemon345
+ *          Copyright 2002-2021 The DOSBox Team
```

Comment:
This attribution is incomplete.
The code is based on modem emulation code from Dosbox-Staging not the original Dosbox. Staging has made a number of additions and bugfixes to the original modem code and thus, should be mentioned.